commit 5c87a4226b6374e9caf177922df2a15e6bdd8050
Author: Scott James Remnant <scott@canonical.com>
Date:   Mon Mar 16 15:03:07 2009 +0000

    UBUNTU: SAUCE: (no-up) nbd: Change default partitions per device to 15
    
    This was previously changed by using an "options" line in a modprobe.d
    file, however that practice is now deprecated.  This is because module
    names, option names, their values and even their current defaults can
    all change inside the kernel and module-init-tools has never been kept
    in sync.
    
    In addition, changing the kernel means that the option change will apply
    if the module is built in by users or the OEM team.
    
    Bug: #342563
    
    Signed-off-by: Scott James Remnant <scott@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/drivers/block/nbd.c b/drivers/block/nbd.c
index a9e3980..aa462f7 100644
--- a/drivers/block/nbd.c
+++ b/drivers/block/nbd.c
@@ -81,7 +81,7 @@ static struct dentry *nbd_dbg_dir;
 
 static unsigned int nbds_max = 16;
 static struct nbd_device *nbd_dev;
-static int max_part;
+static int max_part = 15;
 
 /*
  * Use just one lock (or at most 1 per NIC). Two arguments for this:
